docs/contract.yaml

```diff
@@ -0,0 +1,156 @@
+openapi: 3.0.0
+info: 
+  title: Spec Math API Server
+  version: 1.0.0
+  description: An API that exposes some operations from the Spec Math Library
+servers:
+  - description: Spec Math API Server
+    url: http://specmath.cloud.google.com/v1
+paths:
+  /specMath/merge:
+    post:
+      description: |
+      perform a union on two specs, with optional defaults file and conflict resolutions.
+      The endpoint may return a success, which is the result of a successful union,
+      or a failure, which means some unhandled conflicts occurred in the union. In the latter case,
+      a 406 code will be sent along with an array of conflict objects. The user now has two ways
+      to resolve the conflicts. First, they may iterate through the conflicts and add a value to the
+      resolvedValue property based on the options available. They could also manually resolve the
+      conflicts in the original YAML files. In either case, they can send another request to this
+      endpoint with the conflicts resolved.
+      operationId: specMathUnion
```

Comment:
I would just call this "union". If the operationId is used to generate client code, that code would usually be in "SpecMath" package, giving the calling function a name like "SpecMath::union" (if you change this to "union") which seems cleaner than "SpecMath::specMathUnion".